Dockerfile

```diff
@@ -59,12 +59,59 @@ FROM --platform=$BUILDPLATFORM alpine AS releaser
 WORKDIR /work
 ARG TARGETPLATFORM
 RUN --mount=from=binaries \
-  --mount=type=bind,source=/tmp/.version,target=/tmp/.version,from=buildx-version \
+  --mount=type=bind,source=/tmp/.version,target=/tmp/.version,from=version \
   mkdir -p /out && cp buildx* "/out/buildx-$(cat /tmp/.version).$(echo $TARGETPLATFORM | sed 's/\//-/g')$(ls buildx* | sed -e 's/^buildx//')"
 
 FROM scratch AS release
 COPY --from=releaser /out/ /
 
+# Deb
+FROM --platform=$BUILDPLATFORM debian:bullseye-slim AS build-deb
+RUN apt-get update && apt-get install -y debhelper dh-make
+WORKDIR /build
+COPY --from=xx / /
+COPY --from=build /usr/bin/buildx ./docker-buildx-standalone/usr/bin/
+COPY --from=build /usr/bin/buildx ./docker-buildx-plugin/usr/libexec/docker/cli-plugins/docker-buildx
+ARG TARGETPLATFORM
+RUN --mount=type=bind,source=/tmp/.version,target=/tmp/.version,from=version <<EOT
+#!/usr/bin/env bash
+set -e
+version=$(cat /tmp/.version)
+if [[ $version =~ ^[a-f0-9]{7}$ ]]; then
+  version="v0.0.0+${version}"
+fi
+mkdir -p ./docker-buildx-standalone/DEBIAN
+cat > ./docker-buildx-standalone/DEBIAN/control <<EOL
```

Comment:
That seems a bit too much data even for heredocs. Maybe create a template file under contrib or dist.

Reply:
Switched to nfpm so we can handle deb and rpm packages with the same config. Maybe apk is not necessary as buildx is already available for alpine: https://pkgs.alpinelinux.org/packages?name=*buildx*&branch=v3.14